app/helpers/__init__.py

```diff
@@ -1,7 +1,9 @@
 from .init_new_dataset import init_new_dataset
+from .maintain import maintain
 
 
 __all__ = (
     # init_new_dataset
     init_new_dataset,
+    maintain,
 )
```

Comment:
I think the `maintain` function should actually live as a member function on the `ClimateCabinetDBManager` class in `app/__init__.py`. This feels like core functionality to me, and that's where the core functionality lives!

Reply:
I started to try to move maintain into ClimateCabinetDBManager as you suggested, but I'm now not sure if that's right.     ClimateCabinetDBManager appears to as the name indicates be deeply related with the database and won't even instantiate without trying to connect to one. I noted that the fetch and clean stuff is also not in this class, and since maintain (currently) only deals with fetch and clean, it seems to make sense to me to leave it outside ClimateCabinetDBManager instead of clobbering ClimateCabinetDBManager to make it instantiatable without a db.

What do you think?

Reply:
Ah, you know, you're totally right. Let's keep in it `app/helpers` for now. I'm realizing more and more there's kinda two different applications living inside this repo - one that runs data pipelines (everything `fetch` and `clean`) and one that spins up databases. ClimateCabinetDBManager definitely belongs to the latter, and this `maintain` function definitely feels like core functionality of the former, probably why they don't mix exactly.